### PR TITLE
feat: sfs add documents

### PR DESCRIPTION
**Pull request by deer-hang**
add sfsplugin documents.
modify sfs external-plugin version

### CODE REVIEW

File: docs/sfs/sfs.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Huawei Cloud SFS CSI Driver
+
+The SFS CSI Driver is a CSI Specification compliant driver used by Container Orchestrators to manage
+the lifecycle of Huawei Cloud SFS Volumes.
```
Comment (chengxiangdong) [inline docs/sfs/sfs.md:4]: s/Huawei Cloud SFS Volumes/Huawei Cloud SFS

File: docs/sfs/sfs.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+## Compatibility
+
+For sidecar version compatibility, please refer compatibility matrix for each sidecar here
+-https://kubernetes-csi.github.io/docs/sidecar-containers.html.
+
+## Support version
+
+| CSI version | SFS CSI Plugin Version | Kubernetes Version Tested | Features           |
```
Comment (chengxiangdong) [inline docs/sfs/sfs.md:13]: s/SFS CSI Plugin Version/Driver

File: docs/sfs/sfs.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+- Create RBAC resources
+
+```
+kubectl apply -f https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/deploy/sfs-csi-plugin/kubernetes/rbac-csi-sfs-controller.yaml
+kubectl apply -f https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/deploy/sfs-csi-plugin/kubernetes/rbac-csi-sfs-node.yaml
+kubectl apply -f https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/deploy/sfs-csi-plugin/kubernetes/rbac-csi-sfs-secret.yaml
+```
+
+- Install Huawei Cloud SFS CSI Plugin
```
Comment (chengxiangdong) [inline docs/sfs/sfs.md:50]: s/Plugin/driver

File: docs/sfs/sfs.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+...
+kube-system    pod/csi-sfs-controller-65f7488778-zmqpp      4/4     Running   0          50s
+kube-system    pod/csi-sfs-node-9qckv                       3/3     Running   0          36s
+kube-system    pod/csi-sfs-node-n5chb                       3/3     Running   0          36s
+kube-system    pod/csi-sfs-node-wl4p4                       3/3     Running   0          36s
+```
+
+## Examples
+
+**Shares Volume:** [evs share](sfs-share.md)
```
Comment (chengxiangdong) [inline docs/sfs/sfs.md:72]: how about the following:
`
**SFS Shares:** [example](sfs-share.md)
`

File: docs/sfs/sfs.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+## Compatibility
+
+For sidecar version compatibility, please refer compatibility matrix for each sidecar here
+-https://kubernetes-csi.github.io/docs/sidecar-containers.html.
+
+## Support version
+
+| CSI version | SFS CSI Plugin Version | Kubernetes Version Tested | Features           |
+| ----------- | ---------------------- | ------------------------- | ------------------ |
+| v1.5.0      | v0.1.0                 | v1.20 v1.21 v1.22 v1.23   | volume accessRules |
```
Comment (chengxiangdong) [inline docs/sfs/sfs.md:15]: s/volume accessRules/shares

File: docs/sfs/sfs.md (hunk 1 above)
Comment (deer-hang) [reply docs/sfs/sfs.md:4]: done

Comment (deer-hang) [reply docs/sfs/sfs.md:4]: done

File: docs/sfs/sfs.md (hunk 2 above)
Comment (deer-hang) [reply docs/sfs/sfs.md:13]: done

File: docs/sfs/sfs.md (hunk 3 above)
Comment (deer-hang) [reply docs/sfs/sfs.md:50]: done

File: docs/sfs/sfs.md (hunk 4 above)
Comment (deer-hang) [reply docs/sfs/sfs.md:72]: done

**Pull request merged**